src/app/api/v1/user/scan-result/[onerepScanResultId]/resolution/route.ts

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getServerSession } from "next-auth";
+import { NextRequest, NextResponse } from "next/server";
+
+import { authOptions } from "../../../../../utils/auth";
+import {
+  isOnerepScanResultForSubscriber,
+  markOnerepScanResultAsResolved,
+} from "../../../../../../../db/tables/onerep_scans";
+
+export type ResolveScanResultResponse =
+  | {
+      success: true;
+    }
+  | { success: false; message: string };
+
+export async function POST(
```

Comment:
Hmm, so if I'm understanding correctly, here's how manual resolution works:
1) the frontend would probably request the latest scan results for a profile at some point
2) with the list of scan results (one per data broker), we can essentially get the list of `onerepScanResultId` on the frontend
3) in the UI, we can resolve manually per data broker, with this POST call, we simply pass in the id

Reply:
Yep, that's exactly it! Specifically, [this UI](https://www.figma.com/file/CaEKIhvSJqf6KNIMzSkt40/Concepts-for-Monitor-MVP-Redesign?type=design&node-id=1290-62939&mode=dev) has "Mark as fixed" buttons that would send this POST call.

What's the edge case that you mentioned?